### PR TITLE
Feat/sprint_3_4/add_logic_get_animal_info

### PR DESCRIPTION
**Pull request by Samwa-ua**
Fixed some errors from previous feat
Implemented client logic for fetching data from animals DB
Created some temp routes for easy navigation to the pages with corresponding info

### CODE REVIEW

**Review by Andriy445 — APPROVED** [review]

**Review by andrii-dziu — APPROVED** [review]

File: client/src/app/components/animal-list-card/animal-info.service.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { IAnimalsInfo } from 'src/app/interfaces/animals';
+import { map } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AnimalInfoService {
+  private animalsUrl = 'http://localhost:3200/api/animals';
```
Comment (vladtymo) [inline client/src/app/components/animal-list-card/animal-info.service.ts]: Should we have this constant value in the service class?

File: client/src/app/components/animal-list-card/animal-list-card.component.html [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
+          alt="image of pet"
+        />
+        <mat-card-header>
+          <mat-card-title>{{ animal.name }}</mat-card-title>
+        </mat-card-header>
+        <mat-card-content>
+          <mat-card-subtitle
+            >Age:
+            <span class="main_info">{{ animal.age }} </span></mat-card-subtitle
+          >
```
Comment (vladtymo) [inline client/src/app/components/animal-list-card/animal-list-card.component.html]: Is it a good code formatting style of tags?

File: client/src/app/components/animal-list-card/animal-list-card.component.ts [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
 })
 export class AnimalListCardComponent implements OnInit {
+  animalsInfo: any = [];
 
-  constructor() { }
-
-  ngOnInit(): void {
+  constructor(private animalInfo: AnimalInfoService) {
+    this.animalInfo.getAnimalsInfo().subscribe((item) => {
+      console.log(item);
```
Comment (vladtymo) [inline client/src/app/components/animal-list-card/animal-list-card.component.ts:14]: I suppose we should't use console.log here...

File: client/src/app/components/header/header.component.html [hunk 4]
```diff
@@ -13,8 +13,8 @@
       <ul class="header__nav">
         <li><button mat-button routerLink="/">Home</button></li>
         <li><button mat-button>About</button></li>
-        <li><button mat-button>Pets</button></li>
-        <li><button mat-button routerLink="/detailInfo">Vets</button></li>
+        <li><button mat-button routerLink="/animalsList">Pets</button></li>
+        <li><button mat-button routerLink="/vetsList">Vets</button></li>
         <li><button mat-button>Contacts</button></li>
       </ul>
       <a class="header__phone" href="">+38(096)-937-99-92</a>
```
Comment (vladtymo) [inline client/src/app/components/header/header.component.html]: Why the attribute href is empty?

File: client/src/app/components/vet-list-card/vet-info.service.ts [hunk 5]
```diff
@@ -1,30 +1,26 @@
...
   providedIn: 'root',
 })
 export class VetInfoService {
-  httpOptions = {
-    headers: new HttpHeaders({ 'Content-Type': 'application/json' }),
-  };
-
   private vetDetailsUrl = 'http://localhost:3200/api/vet';
-  private veServicesUrl = 'http://localhost:3200/api/services';
+  private vetServicesUrl = 'http://localhost:3200/api/services';
```
Comment (vladtymo) [inline client/src/app/components/vet-list-card/vet-info.service.ts]: Consider moving the server path outside to reuse then later.

File: client/src/app/interfaces/animals.ts [hunk 6]
```diff
@@ -0,0 +1,7 @@
+export interface IAnimalsInfo {
+  id: Number;
```
Comment (vladtymo) [inline client/src/app/interfaces/animals.ts]: Why this type name starts with upper char while the others with lower?

**Review by vladtymo — CHANGES_REQUESTED** [review]

File: client/src/app/components/animal-list-card/animal-list-card.component.html (hunk 2 above)
Comment (Samwa-ua) [reply client/src/app/components/animal-list-card/animal-list-card.component.html:17]: Have no idea how to solve it yet...

**Review by vladtymo — APPROVED** [review]

**Pull request merged**